<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageTerminal</class>
 <widget class="QWidget" name="PageTerminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextBrowser" name="terminalBrowser">
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
      </font>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Send help command to VESC</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/res/icons/Help-96.png</normaloff>:/res/icons/Help-96.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HistoryLineEdit" name="terminalEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Send command</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/res/icons/Send File-96.png</normaloff>:/res/icons/Send File-96.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Clear terminal</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/res/icons/Delete-96.png</normaloff>:/res/icons/Delete-96.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoryLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/historylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>PageTerminal</receiver>
   <slot>clearTerminal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>terminalEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>sendButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>571</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clearTerminal()</slot>
 </slots>
</ui>
